telegram/user.py

```diff
@@ -646,13 +646,18 @@ def send_invoice(
         timeout: ODVInput[float] = DEFAULT_NONE,
         api_kwargs: JSONDict = None,
         allow_sending_without_reply: ODVInput[bool] = DEFAULT_NONE,
+        max_tip_amount: int = None,
+        suggested_tip_amounts: List[int] = None,
     ) -> 'Message':
         """Shortcut for::
 
             bot.send_message(update.effective_user.id, *args, **kwargs)
 
         For the documentation of the arguments, please see :meth:`telegram.Bot.send_invoice`.
 
+        .. versionchanged:: 13.5
+            As of Bot API 5.2, the parameter :attr:`start_parameter` is optional.
+
```

Comment:
maybe also mention the new attributes here?

Reply:
personally, I'd say that's covered in `For the documentation of the arguments, please see :meth:`telegram.Bot.send_invoice`.` …

Reply:
Thats about what the arguments do. I would still add new ones or order changes there, since arguments are a important part of the shortcuts.

Reply:
I will add the warnings, but not the new parameters - if people want a list of all available arguments, they should have a look at the bot method docs. otherwise we might just as well repeat the complete argument list in all shortcuts and give ourselfs more docstrings to maintain.